Expo1900/Expo1900/Controller/DetailViewController.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  DetailViewController.swift
+//  Expo1900
+//
+//  Created by 재재, 보리사랑 on 2022/06/17.
+//
+
+import UIKit
+
+class DetailViewController: UIViewController {
```

Comment:
```suggestion
final class DetailViewController: UIViewController {
```
=> 이건 지금은 정말 모르셔도 되는데 final 을 붙이면 성능적으로 좀 더 효율적입니다. 
지금 단계에서는 모르셔도 되는 부분이 이니 그런가 보다 하고 넘어가시면 될 듯 합니다 :)

> 참고 자료

* [swift-optimizationTips](https://github.com/apple/swift/blob/main/docs/OptimizationTips.rst#advice-use-final-when-you-know-the-declaration-does-not-need-to-be-overridden)
* [스위프트 성능 이해하기-유용하](https://www.youtube.com/watch?v=z1Gf6EosaUQ)